cypress/integration/favorites_list_spec.js

```diff
@@ -8,7 +8,8 @@
 describe('Favorites List Tests', () => {
 
 
-    let viewports = [Cypress.env('desktop'),Cypress.env('tablet'),Cypress.env('mobile')];
+    // let viewports = [Cypress.env('desktop'),Cypress.env('tablet'),Cypress.env('mobile')];
```

Comment:
@trigal2012 remember to re-enable testing on all viewports

Reply:
they are enabled. I think you are looking at an older file. This one shows "Outdated".